pkg/cmd/cli/backup/create.go

```diff
@@ -256,6 +256,7 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 
 func (o *CreateOptions) BuildBackup(namespace string) (*velerov1api.Backup, error) {
 	backupBuilder := builder.ForBackup(namespace, o.Name)
+	fmt.Printf("%+v\n", backupBuilder.Result())
```

Comment:
Should this be removed before final merge?

Reply:
😅 yep, thanks